scale/product/test/utils.py

```diff
@@ -28,8 +28,9 @@ def create_file_link(ancestor=None, descendant=None, job=None, job_exe=None, rec
                                            recipe=recipe)
 
 
-def create_product(job_exe=None, workspace=None, has_been_published=False, file_name='my_test_file.txt',
-                   file_path='/file/path/my_test_file.txt', media_type='text/plain', file_size=100, countries=None):
+def create_product(job_exe=None, workspace=None, has_been_published=False, is_published=False, uuid='',
```

Comment:
I think it would be better for debugging to default uuid=None and then set a more realistic value:
if not uuid:
    builder = hashlib.md5()
    builder.update(job_exe.job.job_type.id, file_name)
    uuid = builder.hashdigest()